src/config/config_check.c

```diff
@@ -24,60 +25,104 @@ __wt_config_check(
      */
     return (config == NULL || entry->checks == NULL ?
         0 :
-        config_check(session, entry->checks, entry->checks_entries, config, config_len));
+        __config_check(
+          session, entry->checks, entry->checks_entries, entry->checks_jump, config, config_len));
 }
 
 /*
- * config_check_search --
+ * __config_check_compare --
+ *     Compare function used for binary search.
+ */
+static int
+__config_check_compare(const void *keyvoid, const void *checkvoid)
```

Comment:
From memory we needed `WT_CDECL` on any callbacks we pass to standard library calls, otherwise MSVC does something weird related to calling conventions (sorry I'm so vague on the details, it's been a while since I looked at this).

Reply:
Thanks, I fixed that.